### ISSUE TITLE
When interacting with elements in the category filter, Android Voice Assistant (text-to-speech) focus is stolen by "multi-page view"

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#121]
## Description of issue

When you have the Voice Assistant enabled on Android, interacting with the Category filter is a clunky experience. The focus of the tap is constantly stolen by "multi-page view", requiring multiple taps before you can finally read out the selection you wish to hear. 

## Expected behaviour

When tapping, the tapped element should be successfully highlighted and read out.

## Steps to reproduce

- Enable the Voice Assistant (Settings > Accessibility > Voice > Voice Assistant)
- Open the Events page in the Pride App
- Attempt to tap/double-tap on a category 
> It initially selects (and highlights) the category before being interrupted - the whole page is highlighted and it reads out "multi-page view, Show me, 0 selected"
- Tap on the same category
> It is selected and read out correctly

It works the same way when trying to interact with the Cancel/Clear All buttons. 

## OS / Browser / Device model

Samsung Galaxy S8 (Android 8.0)

**Comment by declanslevin** [comment]
After a quick re-test it looks like this isn't happening anymore, so I'll give it some more thorough testing and close if I can't reproduce it
